rdrf/registry/patients/migrations/0060_patient_search_index.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.16 on 2023-02-10 08:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('patients', '0059_living_status_column_permission'),
+        ('rdrf', '0152_apply_unaccent_extension'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+            CREATE FUNCTION sanitise_text_for_search(text varchar) RETURNS text LANGUAGE SQL IMMUTABLE AS
```

Comment:
I was able to get the error to disappear by changing this statement:

```diff
diff --git a/rdrf/registry/patients/migrations/0060_patient_search_index.py b/rdrf/registry/patients/migrations/0060_patient_search_index.py
index 9daef3200..6f35cdd3d 100644
--- a/rdrf/registry/patients/migrations/0060_patient_search_index.py
+++ b/rdrf/registry/patients/migrations/0060_patient_search_index.py
@@ -13,16 +13,18 @@ class Migration(migrations.Migration):
     operations = [
         migrations.RunSQL(
             sql="""
-            CREATE FUNCTION sanitise_text_for_search(text varchar) RETURNS text LANGUAGE SQL IMMUTABLE AS
-            'SELECT unaccent(REPLACE(text, '''''''', ''''));'
+            CREATE FUNCTION sanitise_text_for_search(text varchar) RETURNS text AS
+            $func$
+            SELECT unaccent(REPLACE(text, '''', ''));
+            $func$ LANGUAGE sql IMMUTABLE;
             """,
             reverse_sql="DROP FUNCTION sanitise_text_for_search;"
         ),
         migrations.RunSQL(
             sql='''
             CREATE INDEX patient_family_name_idx ON patients_patient 
-            USING gin ((to_tsvector('simple'::regconfig, 
-            COALESCE(sanitise_text_for_search("family_name"), '') || ' ' || COALESCE(sanitise_text_for_search("given_names"), ''))));
+            USING gin (to_tsvector('simple'::regconfig, 
+            COALESCE(sanitise_text_for_search(family_name), '') || ' ' || COALESCE(sanitise_text_for_search(given_names), '')));
             ''',
             reverse_sql="DROP INDEX patient_family_name_idx;"
         )
```

Unfortunately, the explain still doesn't seem to be using the index, even with hundreds of thousands of patients

schema.py:614
```
Unique  (cost=44951.71..45009.31 rows=480 width=3993)
  ->  Sort  (cost=44951.71..44952.91 rows=480 width=3993)
        Sort Key: patients_patient.family_name, patients_patient.given_names, patients_patient.date_of_birth, patients_patient.id, patients_patient.consent, patients_patient.consent_clinical_trials, patients_patient.consent_sent_information, patients_patient.consent_provided_by_parent_guardian, patients_patient.maiden_name, patients_patient.umrn, patients_patient.date_of_death, patients_patient.place_of_birth, patients_patient.date_of_migration, patients_patient.country_of_birth, patients_patient.ethnic_origin, patients_patient.sex, patients_patient.home_phone, patients_patient.mobile_phone, patients_patient.work_phone, patients_patient.email, patients_patient.next_of_kin_family_name, patients_patient.next_of_kin_given_names, patients_patient.next_of_kin_relationship_id, patients_patient.next_of_kin_address, patients_patient.next_of_kin_suburb, patients_patient.next_of_kin_state, patients_patient.next_of_kin_postcode, patients_patient.next_of_kin_home_phone, patients_patient.next_of_kin_mobile_phone, patients_patient.next_of_kin_work_phone, patients_patient.next_of_kin_email, patients_patient.next_of_kin_parent_place_of_birth, patients_patient.next_of_kin_country, patients_patient.active, patients_patient.inactive_reason, patients_patient.user_id, patients_patient.carer_id, patients_patient.living_status, patients_patient.patient_type, patients_patient.stage_id, patients_patient.created_at, patients_patient.last_updated_at, patients_patient.last_updated_overall_at, patients_patient.created_by_id, (unaccent(replace((patients_patient.given_names)::text, ''''::text, ''::text))), (unaccent(replace((patients_patient.family_name)::text, ''''::text, ''::text))), (to_tsvector('simple'::regconfig, ((COALESCE(unaccent(replace((patients_patient.given_names)::text, ''''::text, ''::text)), ''::text) || ' '::text) || COALESCE(unaccent(replace((patients_patient.family_name)::text, ''''::text, ''::text)), ''::text))))
        ->  Gather  (cost=1000.42..44930.33 rows=480 width=3993)
              Workers Planned: 2
              ->  Nested Loop  (cost=0.42..43805.33 rows=200 width=3993)
                    ->  Parallel Seq Scan on patients_patient  (cost=0.00..42308.50 rows=200 width=3897)
                          Filter: (active AND (upper((to_tsvector('simple'::regconfig, ((COALESCE(unaccent(replace((given_names)::text, ''''::text, ''::text)), ''::text) || ' '::text) || COALESCE(unaccent(replace((family_name)::text, ''''::text, ''::text)), ''::text))))::text) ~~ (('%'::text || upper(replace(replace(replace(unaccent('cardi'::text), '\'::text, '\\'::text), '%'::text, '\%'::text), '_'::text, '\_'::text))) || '%'::text)))
                    ->  Index Scan using patients_patient_rdrf_registry_patient_id_11425044 on patients_patient_rdrf_registry  (cost=0.42..7.21 rows=1 width=4)
                          Index Cond: (patient_id = patients_patient.id)
                          Filter: (registry_id = 2)
```

Reply:
I've removed the migration for the index and fixed the loader CSS on the latest commit.